server/game/Deck.ts

```diff
@@ -14,6 +15,7 @@ export class Deck {
             deckCards: [] as Card[],
             outOfPlayCards: [],
             outsideTheGameCards: [] as Card[],
+            tokens: [] as TokenCard[],
```

Comment:
Yeah this is not named well right now - the `Deck` class is not actually how cards are stored on the player, it's an API model that's used to receive the card definitions for the game from the server (equivalent the current card json files in the test folder).

The token card data will need to be provided in every game since the tokens can be generated and aren't included in the deck.